src/main/java/uk/gov/hmcts/sscs/json/RoboticsJsonGenerator.java

```diff
@@ -0,0 +1,139 @@
+package uk.gov.hmcts.sscs.json;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import org.json.JSONObject;
+import org.json.simple.JSONArray;
+import uk.gov.hmcts.sscs.domain.wrapper.*;
+import uk.gov.hmcts.sscs.model.robotics.RoboticsWrapper;
+
+public final class RoboticsJsonGenerator {
+
+    private RoboticsJsonGenerator() {
+    }
+
+    public static JSONObject create(RoboticsWrapper wrapper) {
+        SyaCaseWrapper appeal = wrapper.getSyaCaseWrapper();
+        JSONObject obj = new JSONObject();
+
+        obj = buildAppealDetails(obj, appeal);
+
+        obj.put("caseId", wrapper.getCcdCaseId());
+        obj.put("appellant", buildAppellantDetails(appeal.getAppellant()));
+
+        if (appeal.getRepresentative() != null) {
+            obj.put("representative", buildRepresentativeDetails(appeal.getRepresentative()));
+        }
+
+        if (appeal.getSyaHearingOptions() != null) {
+            obj.put("hearingArrangements", buildHearingOptions(appeal.getSyaHearingOptions()));
+        }
+
+        return obj;
+    }
+
+    private static JSONObject buildAppealDetails(JSONObject obj, SyaCaseWrapper appeal) {
+        obj.put("caseCode", "002DD");
+        obj.put("appellantNino", appeal.getAppellant().getNino());
+        //FIXME: To be implemented at a future date as part of another ticket
+        obj.put("appellantPostCode", "Bedford");
+        obj.put("appealDate", LocalDate.now().toString());
```

Comment:
`LocalDate.now()` is a temp value until a future story?
If so, can we set it to something obviously incorrect, so we don't leave this in by mistake.

Reply:
No this isn't a temp value, this is correct